app/components/publish-actions.tsx

```diff
@@ -121,6 +126,8 @@ const Embed = ({ chartWrapperRef, configKey, locale }: PublishActionProps) => {
         iframeHeight={iframeHeight}
         configKey={configKey}
         locale={locale}
+        state={state}
+        isSingleChart={state?.chartConfigs?.length === 1}
```

Comment:
@noahonyejese it looks like the condition is not aligned between user profile and published chart page, see how it was done in another PR: https://github.com/visualize-admin/visualization-tool/pull/1826/files

I think we should have the same logic also here (could be a shared function), maybe the `isSingleChart` can be renamed to "shouldAllowDisablingBorder" to better indicate the intent behind this property?